win32/tests/job_object_helper_int/job_object_helper_int.c

```diff
@@ -92,6 +99,235 @@ TEST_FUNCTION_CLEANUP(cleanup)
 {
 }
 
+
+TEST_FUNCTION(test_job_object_helper_set_job_limits_to_current_process)
+{
+    /* Check that calling job_object_helper_set_job_limits_to_current_process
+    *  1. creates the job object
+    *  2. assigns the current process to the job object
+    *  3. sets the job limits
+    */
+
+    UUID_T job_name_uuid;
+    (void)uuid_produce(job_name_uuid);
+
+    char job_name[64];
+    snprintf(job_name, sizeof(job_name), "job_test_ebs_%" PRI_UUID_T "", UUID_T_VALUES(job_name_uuid));
+    LogInfo("Runnint test with job name: %s...", job_name);
+
+    THANDLE(JOB_OBJECT_HELPER) result = job_object_helper_set_job_limits_to_current_process(job_name, 50, 1);
+    ASSERT_IS_NOT_NULL(result);
+
+    /* Check that the job object was created */
+    HANDLE job_object = OpenJobObjectA(JOB_OBJECT_QUERY, FALSE, job_name);
+    ASSERT_IS_NOT_NULL(job_object, "Failed to open job object");
+
+    /* Query the Job Object to check if it has 1 process associated with it */
+    JOBOBJECT_BASIC_PROCESS_ID_LIST process_id_list;
+    DWORD return_length = 0;
+    BOOL result_query = QueryInformationJobObject(job_object, JobObjectBasicProcessIdList, &process_id_list, sizeof(process_id_list), &return_length);
+    ASSERT_IS_TRUE(result_query, "Failed to query job object");
+    ASSERT_ARE_EQUAL(int, return_length, sizeof(process_id_list), "Failed to query job object");
+    ASSERT_ARE_EQUAL(int, process_id_list.NumberOfAssignedProcesses, 1, "Job object should have 1 process associated with it");
+
+    /* Check that the current process is assigned to the job object */
+    BOOL ret = TRUE;
```

Comment:
>TRUE [](http://example.com/codeflow?start=15&length=4)

Probably you want to leave this unitialized or even better - actually initialize it with FALSE, so if appears changed by the call to IsProcessInJob.